.github/scripts/coverity-scan.sh

```diff
@@ -42,7 +42,18 @@ INSTALL_DIR="/opt"
 # the version of coverity to use
 COVERITY_BUILD_VERSION="${COVERITY_BUILD_VERSION:-cov-analysis-linux64-2023.6.2}"
 
-. packaging/installer/functions.sh
+SCRIPT_SOURCE="$(
```

Comment:
Could you please rename to something more insidefull like CALLING_SCRIPT_PATH or INVOKER_SCRIPT_PATH

```suggestion
CALLING_SCRIPT_PATH="$(
```

Reply:
It’s the location of the script itself, not anything about how it was called or invoked.